dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java

```diff
@@ -466,6 +468,7 @@ protected Item addItem(Context c, List<Collection> mycollections, String path,
         String mapOutputString = null;
 
         System.out.println("Adding item from directory " + itemname);
+        log.debug("adding item from directory " + itemname);
```

Comment:
System.out will go to the bitbucket if this code executes in a Servlet.  I could never find these in any log.